app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -121,9 +121,13 @@ def parse_node(node)
         parse_node_condition(condition)
       end
 
-      # Searching for the underlying instance. At the moment this search strategy
-      # is tested only with OpenStack.
-      vms = VmOpenstack.where(:uid_ems => new_result[:identity_system].downcase)
+      # Establish a relationship between this node and the vm its on (if its in the system)
+      # supported relationships: oVirt and Openstack
+      types = [ManageIQ::Providers::Redhat::InfraManager::Vm.name,
+               VmOpenstack.name]
```

Comment:
Right ... OpenStack refactoring is coming, but generally @matthewd's finds/replaces have been comprehensive.